modules/gdscript/gdscript_vm.cpp

```diff
@@ -84,6 +106,21 @@ static String _get_var_type(const Variant *p_var) {
 
 	return basestr;
 }
+
+void GDScriptFunction::_profile_native_call(uint64_t t_taken, String func_name, String instance_class_name = "") {
```

Comment:
```suggestion
void GDScriptFunction::_profile_native_call(uint64_t p_t_taken, String p_func_name, String p_instance_class_name) {
```
Default arguments are not defined in the source but the header